model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -506,6 +508,32 @@ object WorkflowStatuses {
   case object Unknown extends WorkflowStatus
 }
 
+object SubmissionRetryStatuses {
+  sealed trait RetryStatus extends RawlsEnumeration[RetryStatus] {
+    def filterWorkflow(workflows: Seq[Workflow]) = {
+      this match {
+        case RetryFailed => workflows.filter(wf => wf.status.equals(Failed))
+        case RetryAborted => workflows.filter(wf => wf.status.equals(Aborted))
+        case RetryBoth => workflows.filter(wf => wf.status.equals(Failed) || wf.status.equals(Aborted))
+      }
+    }
+    override def toString = getClass.getSimpleName.stripSuffix("$")
+    override def withName(name: String) = SubmissionRetryStatuses.withName(name)
+  }
+
+  def withName(name: String): RetryStatus = {
+    name match {
+      case "Failed" => RetryFailed
```

Comment:
minor nit that it will help with debugability if enum strings match the value names. For example `case "Failed" => Failed`